ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -157,6 +159,44 @@ public String getVersion() {
         return CompletableFuture.completedFuture(buildRaoResult(bestScore, preOptimVariant, bestResultVariant));
     }
 
+    private SystematicSensitivityAnalysisResult runSensi(Network network,
```

Comment:
Same. runSensitivityComputation ? Or runSystematicSensitivityAnalysis to be consistent with `SystematicSensitivityAnalysisService.runAnalysis`

Reply:
ok